### PR TITLE
Close #541: Add Generic Discourse entry

### LINKED ISSUE #541: Discourse
Discourse is a platform, not a site.

It's pretty much impossible to delete your account, sometimes you can go anonymous (with post history retained), sometimes you can get it deleted by the mods. There isn't really information out there to say this, since there are many Discourse sites, I think we should do something about it.

### PR DESCRIPTION
**Pull request by tupaschoal**
Ping @jtagcat

### CODE REVIEW

**Comment by jtagcat** [comment]
I normally sign in with GH/GL to Discourse, god damn annoying to sign up everywhere.

See https://github.com/settings/applications

**Comment by tupaschoal** [comment]
I see, I'm just listing it as best as we can so people can take informed decisions, not sure they can tell it's a discourse hosted, but anyway.

File: _data/sites.json [hunk 1]
```diff
@@ -1969,6 +1969,20 @@
...
+        "name": "Discourse",
+        "url": "https://www.discourse.org/",
+        "difficulty": "impossible",
+        "notes": "There seems to be no standard way of handling account deletions on discourse-hosted forums.",
+        "domains": [
+            "community.netlify.com",
+            "forum.level1techs.com",
+            "forum.rclone.org",
+            "forum.restic.net",
+            "forum.syncthing.net"
```
Comment (jtagcat) [inline _data/sites.json:1982]: Here's mine:
```suggestion
            "forum.syncthing.net",
            "https://community.containo.us",
            "https://forum.cloudron.io",
            "https://forum.duplicati.com",
            "https://discourse.gnome.org",
            "https://discourse.joplinapp.org"
```

**Comment by jtagcat** [comment]
There is probably 10 ways how to get most of the instances, but I don't know and quick google didn't help, involving the instances pinging back home and plain old search engines.

**Comment by jtagcat** [comment]
Closes #541 
The auto-linking in GH only works in comments.

**Comment by tupaschoal** [comment]
> There is probably 10 ways how to get most of the instances, but I don't know and quick google didn't help, involving the instances pinging back home and plain old search engines.

It's alright, we operate on best effort

> The auto-linking in GH only works in comments.

Works in commit messages too, but it only works when merged :)

**Comment by jtagcat** [comment]
I think us working on at the same time and me editing the suggestion made you miss this: https://github.com/jdm-contrib/jdm/pull/581#discussion_r451933606

Scraped the discourse's glory page.

File: _data/sites.json (hunk 1 above)
Comment (tupaschoal) [reply _data/sites.json:1987]: Can you please suggest them without the leading `https://` and preferably sorted?

**Pull request merged**